CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [4.55.0] - 2024-06-11
```

Comment:
You just need to put the description behind "unreleased", the CI will move and create the tag for you. 👍 

```suggestion
### Added
- Integrate Site Editor with Audit App
```